scripts/CompareFitMinimizers/results_output.py

```diff
@@ -74,14 +80,23 @@ def print_group_results_tables(minimizers, results_per_test, problems_obj, group
         header += "\n\n"
         print(header)
         print (tbl_acc_indiv)
+        rel_col_defs = '../../docs/source/concepts/minimizers_comparison/color_definitions.txt'
```

Comment:
I would be better if a modifiable copy of this file was in scripts\CompareFitMinimizers or subfolder of this directory. This is to decouple the concern of generating the concept pages from that of generating the fit benchmarking tables. Also together with the .txt file need a copy of the accompanying docs\source\_static\color_ranking_definitions.js + some container rst to wrap this color scheme with the rst tables for html generation